include/z64scene.h

```diff
@@ -33,6 +33,11 @@ typedef struct {
     /* 0x0E */ s16   params;
 } TransitionActorEntry; // size = 0x10
 
+typedef struct {
+    /* 0x00 */ u8 numActors;
+    /* 0x04 */ TransitionActorEntry* list;
+} TransitionActorContext; // size = 0x8
```

Comment:
`TransitionActor_InitContext` is in `z_scene.c` but I wouldn't put this struct in here, to keep this header oriented towards the maps data format and not anything regarding engine code